src/include/utils.h

```diff
@@ -44,6 +44,9 @@ extern "C" {
 #define LONG_TIME_LENGHT  16 + 1
 #define UTC_TIME_LENGTH   29 + 1
 
+extern char* excluded_files_names[];
```

Comment:
I meant - move this to `restore.h`, and call it `restore_last_file_names`

Reply:
Oh sorry, I think I made a mistaking while squashing, and some of my commits were not included. Now it's fixed.